Apps/AdminWebClient/src/ClientApp/public/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
```

Comment:
This is an HTML file, how is the BASE_URL swapped out?

Reply:
some docs I found https://developers.google.com/web/fundamentals/primers/service-workers/registration